cstar/case.py

```diff
@@ -535,8 +535,8 @@ def run(
         account_key=None,
         walltime=cstar_system.environment.max_walltime,
         queue=cstar_system.environment.primary_queue,
-        job_name="my_case_run",
-    ) -> None:
+        job_name=None,
+    ) -> Optional["SchedulerJob"]:
```

Comment:
re. this and the above, comment, we do not always have a scheduler. Our integration tests on Github, for example, do not use a scheduler.

Reply:
Okay. Just checking - does this mean that the syntax for running a case becomes different on different systems?

Reply:
No, if `scheduler` is `None` then `Case.run()` will return `None` (and the execution will happen in the user's current python session). If there's a scheduler, it will return an appropriate `SchedulerJob` instance and the execution will happen somewhere else on the system.